lib/ide/src/controller/node.rs

```diff
@@ -99,22 +103,62 @@ impl Interface for Controller {
     }
 
     fn position(&self) -> FallibleResult<Position> {
-        todo!()
+        self.graph.get_module().get_node_position(self.id)
     }
 
-    fn set_expression(&self, _ast:Ast) -> FallibleResult<()> {
-        todo!()
+    fn set_position(&self, new_position:Position) -> FallibleResult<()> {
+        Ok(self.graph.get_module().set_node_position(self.id, new_position))
     }
 
-    fn set_expression_text(&self, _expression:&str) -> FallibleResult<()> {
+    fn set_expression(&self, _ast:Ast) -> FallibleResult<()> {
         todo!()
     }
 
-    fn set_position(&self, _new_position:Position) -> FallibleResult<()> {
+    fn set_expression_text(&self, _expression:&str) -> FallibleResult<()> {
         todo!()
     }
 
     fn subscribe(&mut self) -> Subscriber<controller::notification::Node> {
         todo!()
     }
 }
+
+#[cfg(test)]
+mod test {
+    use enso_prelude::default;
+    use crate::double_representation::graph::Id;
+    use crate::controller;
+    use controller::node::Position;
+    use controller::module;
+    use controller::graph;
+    use controller::graph::Interface;
+    use json_rpc::test_util::transport::mock::MockTransport;
+    use parser::Parser;
+    use uuid::Uuid;
+    use wasm_bindgen_test::wasm_bindgen_test;
+
+    #[wasm_bindgen_test]
+    fn node_operations() {
```

Comment:
I'd suggest creating a code like `"main = bar"` and then you'll have a real node to set position on.